checker/tests/nullness/ExplictTypeVarAnnos.java

```diff
@@ -0,0 +1,22 @@
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+public class ExplictTypeVarAnnos<E extends @Nullable Object> {
+    interface Consumer<A extends @Nullable Object> {}
+
+    public static <B extends @Nullable Object> Consumer<B> cast(
+            final @Nullable Consumer<? super B> consumer) {
+        throw new RuntimeException();
+    }
+
+    public static <C extends @Nullable Object> Consumer<C> getConsumer() {
+        Consumer<@Nullable Object> nullConsumer = null;
+        Consumer<C> result = ExplictTypeVarAnnos.<C>cast(nullConsumer);
+        return result;
+    }
+
+    public Consumer<E> getConsumer2() {
```

Comment:
From the class name and bug fix description I expected an explicit annotation on a type variable declaration - but I don't see one, neither on a declaration nor on a use. Am I missing something?

Reply:
The explicit annotation is on the declaration of `E`.

Reply:
So I thought this was talking about `class C<@Anno T>` - an explicit annotation on the declaration of the type parameter.
Here `E` only has an upper bound. So why call this an "explicit type var annotation"?
To test the lower bound logic, wouldn't you also want an annotation on the type parameter declaration?

Reply:
I meant explicit annotation on the upper or lower bound of a type parameter declaration.  I added tests cases with explicit annotations on the lower bound.